integrations/upgrade/index.test.ts

```diff
@@ -882,17 +877,14 @@ test(
       --- ./src/root.1.css ---
       @import 'tailwindcss/utilities' layer(utilities);
       @import './a.1.css' layer(utilities);
-      @config '../tailwind.config.js';
 
       --- ./src/root.2.css ---
       @import 'tailwindcss/utilities' layer(utilities);
       @import './a.1.css' layer(components);
-      @config '../tailwind.config.js';
 
       --- ./src/root.3.css ---
       @import 'tailwindcss/utilities' layer(utilities);
       @import './a.1.css' layer(utilities);
-      @config '../tailwind.config.js';
       "
     `)
```

Comment:
This whole file only tests the `@import` and `layer()` interop, right? So it's probably fine if this has a config so simple it won't need migration

Reply:
agreed